drivers/SmartThings/matter-evse/profiles/evse-energy-power-meas-energy-mgmt-mode.yml

```diff
@@ -0,0 +1,32 @@
+#EVSE with Electrical Power Measurement & Electrical Energy Measurement Cluster in Electrical Sensor DT and Device Energy Management mode in Energy Management DT
+name: evse-energy-power-meas-energy-mgmt-mode

```

Comment:
this guy (and the file name) should be renamed to `evse-energy-meas-power-meas-energy-mgmt-mode` to fit the standards from the other profiles, no?

Reply:
no, because it is added in the order `-energy`, `-power`, `-meas`, and `-energy-mgmt-mode`.

Reply:
ah, I see. Thanks!